examples/with-antd-mobile/pages/_document.js

```diff
@@ -1,11 +1,30 @@
 import Document, { Head, Main, NextScript } from 'next/document'
 
+// Note that fastclick is here because it is in the antd-mobile docs
+// but it may not be required: https://github.com/ant-design/ant-design-mobile/issues/576
+
 export default class extends Document {
   render () {
     return (
       <html>
         <Head>
-          <script src='/static/hd.min.js' />
+
+          <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no" />
+          <script src="https://as.alipayobjects.com/g/component/fastclick/1.0.6/fastclick.js"></script>
+
+          <style dangerouslySetInnerHTML={{
+            __html: `
+              if ('addEventListener' in document) {
+                document.addEventListener('DOMContentLoaded', function() {
+                  FastClick.attach(document.body);
+                }, false);
+              }
+              if(!window.Promise) {
```

Comment:
Hopefully this issue can be completely avoided though if you feel like we don't need that section at all.

Reply:
You can cut out the Promise section completely since this is already done by next.js

https://github.com/zeit/next.js/blob/965f50beb238eab8aafa9be32833b0e7d2947574/packages/next/client/index.js#L18